jlh/WeatherApp/app/src/main/java/com/e_conomic/weatherapp/domain/model/DomainClasses.kt

```diff
@@ -0,0 +1,9 @@
+package com.e_conomic.weatherapp.domain.model
+
+data class ForecastList(val city: String, val country: String, val dailyForecast: List<Forecast>) {
+    operator fun get(position: Int) = dailyForecast[position]
+    val size: Int
+        get() = dailyForecast.size
+}
+data class Forecast(val date: String, val description: String, val high: Int, val low: Int,
```

Comment:
@tudorgk we think alike - I commented on this as well. Seems GitHub gets confused with overlapping reviews.